.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--
+The RIOT community cares a lot about code quality.
+Therefore, before describing what your contribution is about, we would like
+you to make sure that your modifications are compliant with the RIOT
+coding conventions, see https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions.
+-->
+
+### Contribution description
+
+<!--
+Put here the description of your contribution:
+- describe which part(s) of RIOT is (are) involved
+- if it's a bug fix, describe the bug that it solves and how it is solved
+- you can also give more information to reviewers about how to test your changes
+-->
+
+
+### Issues/PRs references
+
+<!--
+Examples: Fixes #1234. See also #5678. Depends on PR #9876.
+Please use keywords (e.g., fixes, resolve) with link the issues you resolved,
+this way they will be automatically closed when your pull request is merged.
```

Comment:
This sentence still doesn't make sense...

Reply:
not very helpful comment, though 🤔 But @miri64 has a point, the following mind read better:

* Please use keywords (e.g., fixes, resolve) ~~with~~[and] link the issue you resolved [...], *or*
* Please use keywords (e.g., fixes, resolve) with links [to] the issues you resolved [...]

I would also put the Examples below this sentence.

Reply:
@smlng I had similar proposals already in https://github.com/RIOT-OS/RIOT/pull/7650#discussion_r155643015 ;-)

Reply:
oops sorry, hadn't re-read the whole discussion ...